src/app/personside/infotabs/oversikt/OversiktNy.tsx

```diff
@@ -0,0 +1,96 @@
+import { ExternalLinkIcon } from '@navikt/aksel-icons';
+import { BodyLong, ExpansionCard, HGrid, HStack, Heading, Link, VStack } from '@navikt/ds-react';
+import Card from 'src/components/Card';
+
+const menuItems = [
+    {
+        title: 'Oppfølging',
+        href: '/new/person/oppfolging',
+        beskrivelse: 'Kort beskrivelse av hva oppfølging er'
+    },
+    {
+        title: 'Kommunikasjon',
+        href: '/new/person/meldinger',
+        beskrivelse: 'Kort beskrivelse av hva kommunikasjon er'
+    },
+    {
+        title: 'Arbeid',
+        href: '/new/person/arbeid',
+        beskrivelse: 'Kort beskrivelse av hva arbeid er'
+    },
+    {
+        title: 'Utbetaling',
+        href: '/new/person/utbetaling',
+        beskrivelse: 'Kort beskrivelse av hva utbetaling er'
+    },
+    {
+        title: 'Ytelser',
+        href: '/new/person/ytelser',
+        beskrivelse: 'Kort beskrivelse av hva ytelser er'
+    },
+    {
+        title: 'Dokumenter',
+        href: '/new/person/saker',
+        beskrivelse: 'Kort beskrivelse av hva dokumenter er'
+    },
+    {
+        title: 'Varsler',
+        href: '/new/person/varsler',
+        beskrivelse: 'Kort beskrivelse av hva varsler er'
+    }
+];
```

Comment:
```suggestion
] as const;
```

Hvis ikke får vi ikke brukt type-safety fra routeren.